### PR TITLE
Moving the actual broker concerns to a defaultConfig

### PR DESCRIPTION
**Pull request by matzew**
Signed-off-by: Matthias Wessendorf <mwessend@redhat.com>

Better reflect the acutal config is just the global default config

### CODE REVIEW

File: hack/lib/serverless.bash [hunk 1]
```diff
@@ -244,7 +244,8 @@ metadata:
 spec:
   broker:
     enabled: true
-    bootstrapServers: my-cluster-kafka-bootstrap.kafka:9092
+    defaultConfig:
+      bootstrapServers: my-cluster-kafka-bootstrap.kafka:9092
```
Comment (markusthoemmes) [inline hack/lib/serverless.bash:248]: This seems to be a breaking change. How are customers supposed to upgrade? 🤔

Comment (matzew) [reply hack/lib/serverless.bash:248]: Nope, b/c this was introduced a week ago or so, was not there on 1.19 (the `broker` things are new)

Comment (markusthoemmes) [reply hack/lib/serverless.bash:248]: Ahhh, nevermind then, thanks!

**Comment by matzew** [comment]
I updated the CRD... to also have the `defaultConfig` ...

**Comment by matzew** [comment]
/unhold
tested it also end-to-end (via UI from Operator)

**Comment by matzew** [comment]
/assign @pierDipi 
going with the nested `defaultConfig` for the "global" broker settings

**Review by pierDipi — APPROVED** [review]
/lgtm

**Pull request merged**